From: Gordon Ball <gordon@chronitis.net>
Date: Thu, 26 Jan 2017 13:47:34 +0100
Subject: Disable downloading CSS from the web,
 since we can copy it directly from the notebook package instead

Forwarded: Debian-specific
Author: Colin Watson <cjwatson@debian.org>
---
 hatch_build.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/hatch_build.py b/hatch_build.py
index b64ada9..e0afb3d 100644
--- a/hatch_build.py
+++ b/hatch_build.py
@@ -66,6 +66,9 @@ def _get_css_file(template_name, url, filename):
 
 def _get_css_files():
     """Get all of the css files if necessary"""
+    # Disabled in Debian package; we copy them in debian/rules instead.
+    return
+
     in_checkout = osp.exists(osp.abspath(osp.join(here, "..", ".git")))
     if in_checkout:
         print("Not running from git, nothing to do")
